tutorials/darcy_thermo_mech/doc/content/workshop/cpp/02_scope.md

```diff
@@ -18,30 +18,6 @@ Dynamically-allocated (via new) memory *is not* automatically freed at the end o
 
 !---
 
-## "Named" Scopes
```

Comment:
You have to define a scope for the next slide so you could keep that one

Reply:
The names one

Reply:
Most of this will be described verbally in the next slide, the remainder in class